requirements-docs.txt

```diff
@@ -0,0 +1,12 @@
+# Requirements for building the documentation site
+sphinx
+sphinx-gallery
+sphinx_bootstrap_theme
+numpydoc
+sphinx-copybutton
+matplotlib
```

Comment:
I think of this file as "requirements over & above bycycle & its dependencies", and so since mpl is a dependency of bycycle already, I don't think this needs to be noted here.

Reply:
I dropped matplotlib. I didn't notice it was already on the requirements.txt. Thanks!